interface/main/tabs/fullscreen.php

```diff
@@ -0,0 +1,81 @@
+<?php
+Use Esign\Api;
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */

```

Comment:
Look at some of the other files and use the header as a guide and put a header here with license, and copyright information

Reply:
Should I put my name there or just the organization's name?

Reply:
look at how naveen did it in the color picker change in style.php